Makefile

```diff
@@ -43,3 +50,16 @@ test-swift:
 	swift test
 
 test-all: test-linux test-mac test-ios
+
+SDK_PATH = $(shell xcrun --show-sdk-path)
+FRAMEWORKS_PATH = $(SDK_PATH)/System/Library/Frameworks
+COMMON_CRYPTO_PATH = $(FRAMEWORKS_PATH)/CommonCrypto.framework
+COMMON_CRYPTO_MODULE_PATH = $(COMMON_CRYPTO_PATH)/module.map
+define COMMON_CRYPTO_MODULE_MAP
```

Comment:
Whohohoa! Makefile expert! Never seen this before!

Reply:
💅